src/webgl/p5.Camera.js

```diff
@@ -127,9 +127,29 @@ p5.prototype.camera = function (...args) {
  * vertical field of view, aspect ratio (usually width/height), and near and far
  * clipping planes.
  *
- * If no parameters are given, the following default is used:
- * perspective(PI/3, width/height, eyeZ/10, eyeZ*10),
- * where eyeZ is equal to ((height/2) / tan(PI/6)).
+ * If no parameters are given, the default values are used as:
+ *
+ * fov- The default field of view for the camera is such that the full height of renderer is visible when it is positioned at a default distance of 800 units from the camera.
+ *
+ * aspect- The default aspect ratio is the ratio of renderer's width to renderer's height.
+ *
+ * near - The default value for the near clipping plane is 0.1 times the default distance from the camera to the point it is looking at i.e 800.
+ *
+ * far - The default value for the far clipping plane is 10 times the default distance from the camera to the point it is looking at i.e 800.
+ *
+ * If you prefer a fixed field of view, follow these steps:
+ *
+ * 1.Choose your desired field of view angle (`fovy`). This is how wide the camera can see.
```

Comment:
Right now these are getting rendered as paragraphs. I think we want to make these appear as a proper list:
<img width="1053" alt="image" src="https://github.com/processing/p5.js/assets/5315059/249751a9-e14a-4860-a0ba-09539906571f">

To do so, I believe we need a space after the . after each number, and I think we have to take out the blank lines between each list item.